Makefile

```diff
@@ -21,7 +21,7 @@ WEBHOOK_IMG ?= gcr.io/${PROJECT_ID}/webhook:${SHORT_SHA}
 DELETION_DEFENDER_IMG ?= gcr.io/${PROJECT_ID}/deletiondefender:${SHORT_SHA}
 UNMANAGED_DETECTOR_IMG ?= gcr.io/${PROJECT_ID}/unmanageddetector:${SHORT_SHA}
 # Detects the location of the user golangci-lint cache.
-GOLANGCI_LINT_CACHE := $(shell pwd)/.tmp/golangci-lint
+GOLANGCI_LINT_CACHE := /tmp/golangci-lint
```

Comment:
We can use /tmp/k8s-config-connector/golangci-lint  in case other repo also use golangci-lint

Reply:
NIT: I am not sure if we need to do mkdir -p for the cache folder under make lint section at line 98.

Reply:
"__technically__", the golangi-ci lint cache can be shared across repoes and the official docs don't point out a need to separate on a per repo basis. If there's strong opinions I can rename it.

Also re the nit, I don't think we need to mkdir and when I tried it out before the PR that statement was validated so I think we are good.

Reply:
We can change to folder if needed later.